site/shared/connectors/metaMask.md

```diff
@@ -43,8 +43,11 @@ Check out the [MetaMask SDK docs](https://docs.metamask.io/wallet/connect/3rd-pa
 
 `DappMetadata | undefined`
```

Comment:
```suggestion
```

Maybe we can remove these data types if it is confusing? We already mention the fields and default value in the description and provide an example.